packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -527,6 +529,91 @@ void main() {
     await tester.showKeyboard(find.byType(TextField));
     await tester.pump();
   });
+
+  group('getSemanticsData', () {
+    testWidgets('throws when there are no semantics', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        new MaterialApp(
+          home: const Scaffold(
+            body: const Text('hello'),
+          ),
+        ),
+      );
+
+      expect(() => tester.getSemanticsData(find.text('hello')),
+        throwsA(isInstanceOf<StateError>()));
```

Comment:
What's the actual error message here? Does it make clear that the problem can be fixed by enabling semantics?

Reply:
Not sure how to test for it, no semantics enabled has its own error but this just rethrows the error from Iterable.single

Reply:
Updated to throw specific errors